knowledge-base/pass-sqldatasource-columns-as-datasource-for-nested-data-items.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to Use Part of the Columns from a Parent SqlDataSource as DataSource for Nested Data Items
```

Comment:
I would use the words "master-details" in the title, and "Part of the Columns" in the description, so the title would describe the scenario in which this approach can be applied. Something like "How to use a single data set in a master-details scenario" (but with better wording than mine)